src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -8894,19 +8905,18 @@ Real64 CalcEmmelRoof(EnergyPlusData &state,
 
     Real64 Theta; // angle between wind and surface azimuth
 
-    Theta = WindDir - LongAxisOutwardAzimuth - 90.0; // TODO double check theta
-    if (Theta > 180.0) Theta -= 360.0;
+    Theta = CalcWindSurfaceTheta(WindDir, LongAxisOutwardAzimuth);
 
     if (Theta <= 22.5) {
-        Hf = 5.15 * std::pow(WindAt10m, 0.81);
-    } else if ((22.5 < Theta) && (Theta <= 67.5)) {
-        Hf = 3.34 * std::pow(WindAt10m, 0.84);
-    } else if ((67.5 < Theta) && (Theta <= 112.5)) {
-        Hf = 4.78 * std::pow(WindAt10m, 0.71);
-    } else if ((112.5 < Theta) && (Theta <= 157.5)) {
-        Hf = 4.05 * std::pow(WindAt10m, 0.77);
-    } else if ((157.5 < Theta) && (Theta <= 180.0)) {
-        Hf = 3.54 * std::pow(WindAt10m, 0.76);
+        Hf = 5.11 * std::pow(WindAt10m, 0.78);
+    } else if (Theta <= 67.5) {
+        Hf = 4.60 * std::pow(WindAt10m, 0.79);
+    } else if (Theta <= 112.5) {
+        Hf = 3.67 * std::pow(WindAt10m, 0.85);
+    } else if (Theta <= 157.5) {
+        Hf = 4.60 * std::pow(WindAt10m, 0.79);
+    } else if (Theta <= 180.0) {
+        Hf = 5.11 * std::pow(WindAt10m, 0.78);
```

Comment:
Looks like a pasteo error. Based on the paper, these are the numbers for a vertical surface, not a roof surface.